Formula/gocr.rb

```diff
@@ -21,7 +21,7 @@ class Gocr < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "4fc5af66eb1d5e1345c1599d57da5a4695f7d37975ba96bfa97dd481ec3a7b1c"
   end
 
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
```

Comment:
Should be fine to migrate, but may not function correctly until we swap keg-only/linkage as I think some code might shell out to `djpeg`.

Reply:
We can wait till just before we're ready to swap linkage then. I think that shouldn't take too long.